up42/catalog.py

```diff
@@ -121,8 +81,8 @@ def construct_parameters(
                 list, GeoDataFrame, Point, Polygon.
             start_date: Query period starting day, format "2020-01-01".
             end_date: Query period ending day, format "2020-01-01".
-            sensors: The satellite sensors to search for, one or multiple of
-                ["pleiades", "spot", "sentinel1", "sentinel2", "sentinel3", "sentinel5p"]
+            collection: The satellite sensor collection to search for, e.g. "PHR",
```

Comment:
@mykolakozyr @iliascholl-up42 @vzup42 @dc-up42 Seeing the "Catalog Console: Search by Hosts"  story on productboard, does it make sense to already implement it in the SDK as this, a single collection name string instead of a list of collections? Or will the API still accept a list of collections after? Would prefer to not change the parameter type if this changes in a few weeks.

Reply:
After discussion with Mykola, we will keep it as a list.